src/reducers/tests/transactionReducer.test.ts

```diff
@@ -189,13 +195,19 @@ describe('transactionReducer', () => {
         payloadEstimatedFeeLoading,
         {} as SourceTargetState,
         () => 'type',
-        true
+        true,
+        null,
+        null,
+        9
```

Comment:
With so many parameters and some of them being optional I'd consider a single configuration object to be passed as the only argument to `setPayloadEstimatedFee` instead of these 9 positional arguments.

Reply:
yeah, completely agree. I think it started with just a few and it kept growing.